x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_errors.tsx

```diff
@@ -56,7 +56,12 @@ export function WorkspaceErrors(props: Props) {
               {activeError.longMessage ? (
                 <>
                   <EuiSpacer />
-                  <EuiText size="s"> {activeError.longMessage as React.ReactNode}</EuiText>
+                  <EuiText size="s">
+                    {' '}
+                    {typeof activeError.longMessage === 'function'
+                      ? activeError.longMessage()
+                      : activeError.longMessage}
```

Comment:
Since there are many usages of this maybe we could extract this logic.

```ts
const unwrapUserMessage = (msg: UserMessage) => typeof msg.longMessage === 'function' ? msg.longMessage() : msg.longMessage;
```

```suggestion
                    {unwrapUserMessage(activeError}
```

Reply:
updated here https://github.com/elastic/kibana/pull/192492/commits/18f4f3bd4a7d11ece8f9f679a49fe8b9a43a4687